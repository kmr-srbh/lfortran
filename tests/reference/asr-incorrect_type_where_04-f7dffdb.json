{
    "basename": "asr-incorrect_type_where_04-f7dffdb",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/incorrect_type_where_04.f90",
    "infile_hash": "42f8ace2a232052937d439ce0f429e27dd6d8ddf373aa4865c19e809",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-incorrect_type_where_04-f7dffdb.stderr",
    "stderr_hash": "f3c67a27e8602ff8aa1a4b211180b254ac508c9e2bfaae36e68ff621",
    "returncode": 2
}