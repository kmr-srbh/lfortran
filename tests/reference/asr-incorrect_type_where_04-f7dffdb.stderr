semantic error: The RHS character len must be known at compile time
 --> tests/errors/incorrect_type_where_04.f90:3:4
  |
3 |    character(5), parameter :: c = "hello"
  |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
